modules/navier_stokes/src/fvkernels/PINSFVMomentumAdvection.C

```diff
@@ -22,6 +22,8 @@ PINSFVMomentumAdvection::validParams()
   params.addClassDescription("Object for advecting superficial momentum, e.g. rho*u_d, "
                              "in the porous media momentum equation");
   params.addRequiredParam<MooseFunctorName>(NS::porosity, "Porosity");
+  // With porosity jumps our pressure stencil can grow to 3
+  params.set<unsigned short>("ghost_layers") = 3;
```

Comment:
Do you think that we could add checks to see if we really need this and if we do we can just adjust the layers? I am not sure how much effort we spend on communication be if you think it is negligible we can roll with this for sure. This is probably more of a problem in 3D than in 1/2D.